### PR TITLE
Create Splash Screen

### PR DESCRIPTION
**Pull request by leimig**
- Creates splash screen to improve user experience while client is loading
- Improve index.html formatting

### CODE REVIEW

File: client/app/index.html [hunk 1]
```diff
@@ -20,400 +20,559 @@
...
+     * to wait for the CSS file to load.
+     */
+    [ng\:cloak].splash,
+    [ng-cloak].splash,
+    [data-ng-cloak].splash,
+    [x-ng-cloak].splash,
+    .ng-cloak.splash,
+    .x-ng-cloak.splash {
+      background-color: #000;
+      background-image: url("assets/img/images/Space.jpg");
```
Comment (luciorubeens) [inline client/app/index.html]: the path has changed to `assets/images/images`

File: client/app/index.html [hunk 2]
```diff
@@ -20,400 +20,559 @@
...
+      margin-left: auto;
+      margin-right: auto;
+      width: 130px;
+    }
+    @keyframes fadein {
+      from { opacity: 0; }
+      to   { opacity: 1; }
+    }
+    /* Firefox < 16 */
+    @-moz-keyframes fadein {
```
Comment (luciorubeens) [inline client/app/index.html]: it will be loaded in chromium, so it is not necessary

**Comment by luciorubeens** [comment]
hmm i liked, but the splash screen appear when switch network

**Comment by leimig** [comment]
Don't you think it should appear when changing networks? When I test it local I get a black screen sometimes when doing it.

**Comment by luciorubeens** [comment]
damn, true that. I had not noticed that black screen

**Comment by luciorubeens** [comment]
we will made a custom splash screen, wait a bit

**Comment by boldninja** [comment]
@leimig this PR will count in October when merged so don't worry you'll receive bounty when merged.

**Comment by luciorubeens** [comment]
+5 👍🏻

merged manually

**Comment by leimig** [comment]
Thanks @luciorubeens

**Pull request closed without merging**